src/spikeinterface/widgets/motion.py

```diff
@@ -7,7 +7,16 @@
 
 class MotionWidget(BaseWidget):
     """
-    Plot unit depths
+    Plot a summary of motion correction outputs. The plots shown are:
+
+    1) 'Peak depth' plot displaying the peak location for all detected spikes over time
+       before motion correction.
+    2) 'Corrected peak depth' displayed the peak location for all detected spikes
+       over time after motion correction.
+    3) 'Motion vectors' plot shows the estimated displacement per channel
```

Comment:
I actually think I got this wrong, it is displacement across binned depths, not channels? In general I can fix these docstring to better use the terminology from the drift correction tutorial.

Reply:
Okay I think I have fixed it now, by replacing "per channel" with "per spatial bin".